apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -160,38 +168,6 @@ const ProfileView = () => {
     }
   };
 
-  const formMethods = useForm({
-    defaultValues: {
-      username: "",
-      avatar: "",
-      name: "",
-      email: "",
-      bio: "",
-    },
-  });
-
-  const {
-    reset,
-    formState: { isSubmitting, isDirty },
-  } = formMethods;
-
-  useEffect(() => {
-    if (user) {
-      reset(
-        {
-          avatar: user?.avatar || "",
-          username: user?.username || "",
-          name: user?.name || "",
-          email: user?.email || "",
-          bio: user?.bio || "",
-        },
-        {
-          keepDirtyValues: true,
-        }
-      );
-    }
-  }, [reset, user]);
```

Comment:
Spots relying on useEffect reinitialize form values. Instead we hold on rendering the form UNTIL initial values are fetched and ready.